mdoc/Mono.Documentation/Frameworks/FrameworkTypeEntry.cs

```diff
@@ -8,6 +8,10 @@ namespace Mono.Documentation
 	class FrameworkTypeEntry : IComparable<FrameworkTypeEntry>
 	{
 		SortedSet<string> members = new SortedSet<string> ();
+		SortedSet<string> memberscsharpsig = new SortedSet<string> ();
+
+		CSharpFullMemberFormatter formatter = new CSharpFullMemberFormatter ();
```

Comment:
Do we really want to use the `CSharpFullMemberFormatter` here, and not the IL formatter?